audeer/core/io.py

```diff
@@ -373,9 +373,9 @@ def list_file_names(
         list of path(s) to file(s)
 
     Example:
-        >>> path = mkdir('path1')
-        >>> open(os.path.join(path, 'file1'), 'a').close()
-        >>> list_file_names(path, basenames=True)
+        >>> dir_path = mkdir('path1')
```

Comment:
Because `path1` gets cleaned already after the end of all tests.

But as there is no reason to use `path1` and `file1` here I changed to `path` and `file`.